examples/android/lib/BUILD.bazel

```diff
@@ -1,7 +1,10 @@
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_compiler_plugin", "kt_android_library")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library", "kt_compiler_plugin")
 
 kt_compiler_plugin(
     name = "serialization_plugin",
+    compile_phase = True,
+    id = "org.jetbrains.kotlin.serialization",
+    stubs_phase = True,
```

Comment:
Does this mean the plugin and the code it generates is run twice, once during KAPT and once during regular kotlinc?

Reply:
Yep. If you don't, the stubs can be bad.

Reply:
So, after discussion, I think we can default these to true, because most cases it's what folks will want. Badly behaved plugins may trip this up, but most users of compiler plugins likely won't understand the dynamics, and all of the existing corpus of plugins seems to be safe for these as a default.